java-tools/OsmAndServer/src/main/resources/application.yml

```diff
@@ -30,6 +30,7 @@ osmand:
   srtm.location: ${SRTM_LOCATION:}
   weather.location: ${WEATHER_LOCATION:}
   wiki.location: ${osmand.files.location}/wikidata/wikidata_commons_osm.sqlitedb
+  android.translations.location: ${osmand.files.location}/android/OsmAnd/res/
```

Comment:
osmand.files.location: /var/www-download

there is no `/android/OsmAnd/res/` in the specified directory

please think another way to get the resources and refresh them regularly

possible path is workspace of Jenkins WebSite_UpdateFrontend job (depened to Web updates...)